.travis.yml

```diff
@@ -4,6 +4,7 @@ elixir:
   - 1.4.5
   - 1.5.3
   - 1.6.6
+  - 1.7.0-rc.0
```

Comment:
I saw rc 1 came out yesterday: https://elixirforum.com/t/elixir-v1-7-0-rc-1-is-out/15221
Should we update it to test against that?

Reply:
Could not get it to work on 1.7.0.rc.1. Something about a missing package for OTP 19: https://travis-ci.org/appsignal/appsignal-elixir/jobs/407046279

```
Installing Elixir 1.7.0-rc.1
0.05s$ wget https://repo.hex.pm/builds/elixir/v1.7.0-rc.1-otp-19.zip
--2018-07-23 09:06:28--  https://repo.hex.pm/builds/elixir/v1.7.0-rc.1-otp-19.zip
Resolving repo.hex.pm (repo.hex.pm)... 151.101.22.2, 2a04:4e42:5::514
Connecting to repo.hex.pm (repo.hex.pm)|151.101.22.2|:443... connected.
HTTP request sent, awaiting response... 403 Forbidden
2018-07-23 09:06:28 ERROR 403: Forbidden.
The command "wget https://repo.hex.pm/builds/elixir/v1.7.0-rc.1-otp-19.zip" failed and exited with 8 during .
```